pages/blog/july-december-2021-at-asyncapi.md

```diff
@@ -0,0 +1,171 @@
+---
+title: 2nd half of 2021 at AsyncAPI
+date: 2021-12-17T06:00:00+01:00
+type: Communication
+tags:
+  - Project Status
+cover: /img/posts/july-december-2021-at-asyncapi/cover.webp
+authors:
+  - name: Lukasz Gornicki
+    photo: /img/avatars/lpgornicki.webp
+    link: https://twitter.com/derberq
+    byline: AsyncAPI Maintainer and Community Guardian
+excerpt: It has been a while since the last update on the things at AsyncAPI Initiative. Lots of things have happened since June. You better have a read or listen to the podcast. 
+featured: true
+---
+
+Around August, I communicated that because vacation is a dead season (in Poland we call it cucumber time) I will not do my monthly updates and write a single vacation summary.
```

Comment:
```suggestion
Around August, I communicated that because vacation is a dead season (in Poland we call it cucumber time) I will not do my monthly updates or write a single vacation summary.
```